### ISSUE TITLE
[Announce] Official support is now Ubuntu 14.04 (Trusty) - ROS Indigo

### ISSUE THREAD

**Issue by 130s** [start-jsk/rtmros_hironx#396]
Dear Kawada's opensource version of Hiro / NEXTAGE (hironxo) users,

tl;dr Now we offer official support for ROS-Indigo.

It's been awhile since [I forenoticed](https://groups.google.com/d/msg/rtm-ros-robotics/qrDWvnjxrV0/RM7tZZ4yfukJ) that the support for hironxo would be switched from ROS Hydro to Indigo. Now that Hydro is officially end-of-lifed by OSRF, we've been cooperating with Kawada, and [all the blockers are finally cleared](https://github.com/start-jsk/rtmros_hironx/issues/202), I'd like to announce our official support target is now Ubuntu 14.04 + ROS Indigo.

ROS Indigo is LTS, or Long Term Support, meaning that you'll keep getting [updates until 2019](http://wiki.ros.org/Distributions). This doesn't mean that all ROS packages will keep updated until then, but I would assume the core ROS libraries will, and by community's good will many non-core libraries too.
In Indigo almost 2,000 ROS packages are available (while 1,700 for Hydro).

Note that this announcement is related only to software on Ubuntu side (VisionPC). Software on QNX (ControllerBox) is out of discussion this time, unless you want the latest `hrpsys` features (e.g. trajectory interruption)*1. 

You have two choices:

**A. Those who want to switch to new Ubuntu 14.04 + ROS Indigo**

Steps:
1. Upgrade Ubuntu to 14.04. Follow http://askubuntu.com/a/203302/24203, and be sure to make any backup copy you may need (the link also helps what to backup from Ubuntu perspective).
   - Also, for Hiro / NEXTAGE perspective, do in particular:
     - Make a backup for `/home/nxuser/Desktop/NxSupervisor` (exact path may be different, but it's the file you see on `Desktop`)
     - Obtain the version of the cameras (IDS ueye). You may get the version by looking at `/usr/lib/libueye_api64.so.X.XX` (X.XX is the version)
2. Install ROS Indigo. Follow http://wiki.ros.org/indigo/Installation/Ubuntu
3. Install Hiro/NX Open software. Follow http://wiki.ros.org/rtmros_nextage/Tutorials/Install%20NEXTAGE%20OPEN%20software%20on%20your%20machine#Install_for_the_first_time
4. If camera aren't working, download and install the driver from [IDS website](https://en.ids-imaging.com/download-ueye-lin64.html) with corresponding version you obtained earlier.

**B. Those who like to stick to 12.04 + Hydro**

Alternatively, you can keep using this combination. It's just you won't get any update via that simple apt-get command. You instead need to obtain and compile source in order to apply the changes.

Step: http://wiki.ros.org/rtmros_nextage/Tutorials/Advanced%20way%20to%20install%20Hironx-NEXTAGE%20OPEN%20software#Install_via_source

If you see any issues/questions, please add comments here.

Your dual-arm robot support team

*1 If this may be your case please contact professional support provider. E.g. TORK (info[a t]opensource-robotics.tokyo.jp).

**Comment by 130s** [comment]
[Announce] Default branch is now `indigo-devel`. Please have a look at https://github.com/start-jsk/rtmros_hironx/issues/367#issuecomment-149943983 for detail.

**Comment by 130s** [comment]
This announcement has served well.
